sourcegraph/sourcegraph
# core workflow: consolidate search results info bar actions

Fixes #38438 

FYI: I decided to duplicate some of the code for easier future removal. I didn't want to leave a big footprint by trying to adjust existing components.

## Demo

https://user-images.githubusercontent.com/6417322/178445655-0aae6a3c-d529-488c-80cb-a1750324d993.mov

## Test plan


* Do a search
* There should be a single menu in the top right corner of the search results, containing:
  * Collapse/Expand search result matches
  * Create monitor
  * Create context
  * Create insight
  * Create batch change
  * Save search

## App preview:

- [Web](https://sg-web-rn-consolidate-search-results.onrender.com)
- [Storybook](https://5f0f381c0e50750022dc6bf7-jykgpbfjee.chromatic.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
